spec/services/discovery/date_builder_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+describe Discovery::DateBuilder do
+  subject { described_class.new(geo_work_presenter) }
+
+  let(:document) { instance_double('Document') }
+  let(:geo_work_presenter) { VectorWorkShowPresenter.new(SolrDocument.new(geo_work.to_solr), nil) }
+
+  describe 'geospatial work document' do
+    before do
+      expect(document).to receive(:layer_modified=)
+      expect(document).to receive(:issued=)
```

Comment:
I'd use `allow` instead of `expect` in the before block. We're stubbing those methods rather than testing if the code calls them.